curriculum/challenges/english/25-front-end-development/lab-golf-score-translator/689f7996426c1534fa5bea66.md

```diff
@@ -0,0 +1,154 @@
+---
+id: 689f7996426c1534fa5bea66
+title: Build a Golf Score Translator
+challengeType: 26
+dashedName: build-a-golf-score-translator
+demoType: onClick
+---
+
+# --description--
+
+In the game of Golf, each hole has a `par`, meaning, the average number of `strokes` a golfer is expected to make in order to sink the ball in the hole to complete the play. Depending on how far above or below `par` your `strokes` are, there is a different nickname.
+
+Your function will be passed `par` and `strokes` arguments. Return the correct string according to this table which lists the strokes in order of priority; top (highest) to bottom (lowest):
+
+<table><thead><tr><th>Strokes</th><th>Return</th></tr></thead><tbody><tr><td>1</td><td>"Hole-in-one!"</td></tr><tr><td>&#x3C;= par - 2</td><td>"Eagle"</td></tr><tr><td>par - 1</td><td>"Birdie"</td></tr><tr><td>par</td><td>"Par"</td></tr><tr><td>par + 1</td><td>"Bogey"</td></tr><tr><td>par + 2</td><td>"Double Bogey"</td></tr><tr><td>>= par + 3</td><td>"Go Home!"</td></tr></tbody></table>
+
+In this lab, you will write a function that converts the `par` and `strokes` to their nickname. 
+
+**Objective:** Fulfill the user stories below and get all the tests to pass to complete the lab.
+
+**User Stories:**
+
+1. You should create a function named `golfScore`.
+1. `golfScore` should take two numeric arguments, which is the par for the course and the amount of strokes made
+1. `golfScore` should return a string.
```

Comment:
these user stories are missing the relation between input and output, instead of in the introduction, please add them here

Reply:
I will add the user stories from the introduction to here.